tsconfig.json

```diff
@@ -1,13 +1,23 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
+    "lib": ["es2015", "dom"],
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "target": "es5",
+    "sourceMap": true
   },
   "include": [
     "src/*.ts",
     "src/**/*.ts",
     "test/*.ts",
-    "system-test/*.ts"
+    "test/**/*.ts",
+    "system-test/*.ts",
+    "browser-test/*.ts",
+    "browser-test/**/*.ts"
+  ],
+  "exclude": [
+    "node_modules",
```

Comment:
You shouldn't need to include node_modules

Reply:
Yes, that's a leftover from something else :)